Documentation/selinux.md

```diff
@@ -1,10 +1,22 @@
 # rkt and SELinux
 
 rkt supports running containers using SELinux [SVirt](http://selinuxproject.org/page/SVirt).
-At start-up, rkt will attempt to read `/etc/selinux/(policy)/contexts/lxc_contexts`.
-If this file doesn't exist, no SELinux transitions will be performed.
-If it does, rkt will generate a per-instance context.
-All mounts for the instance will be created using the file context defined in `lxc_contexts`, and the instance processes will be run in a context derived from the process context defined in `lxc_contexts`.
+The SELinux context for each rkt pod can be set in one of two ways:
 
-Processes started in these contexts will be unable to interact with processes or files in any other instance's context, even though they are running as the same user.
+### Reading SElinux context from the host lxc_contexts file
+
+When invoking `rkt run` or `rkt run-prepared`, rkt will attempt to read `/etc/selinux/(policy)/contexts/lxc_contexts`.
+If this file does not exist, SELinux transitions will be performed.
+Otherwise, rkt will generate a per-pod context.
+All mounts for the pod will be created using the file context defined in `lxc_contexts`, and the processes inside the pod will be run in a context derived from the process context defined in `lxc_contexts`.
+
+Processes started in these contexts will be unable to interact with processes or files in any other pod's context, even though they are running as the same user.
 Individual Linux distributions may impose additional isolation constraints on these contexts - please refer to your distribution documentation for further details.
+
+### Pass the SElinux options
+If `--selinux-options` is provided to `rkt run` or `rkt run-prepared`, then the given SElinux context will be used.
+The flag is a comma separated string in the form of `user:USER,role:ROLE,type:TYPE,level:LEVEL`.
+Users can specify part of the options, e.g. `--selinux-options=user:foo`.
+The context fields defined in `--selinux-options` will override the fields in the context defined in `lxc_contexts`. e.g. if `--selinux-options=user:foo` then the `user` field will be overwriten as `foo`, other fields are not changed.
+
+All process running inside the pod will be created using the provided context, and the process inside the pod will be run in the context.
```

Comment:
This sentence is a bit confusing. What do you mean by the second part, isn't it redundant?